scionlab/hostfiles/scionlab-config

```diff
@@ -298,20 +298,26 @@ def install_scionlab_service(tmpdir):
     scionlab_services = os.path.join(tmpdir, 'scionlab-services.txt')
     if not os.path.exists(scionlab_services):
         _error_exit("Could not find the scionlab-services.txt file in %s", tmpdir)
+    with open(scionlab_services, 'r') as f:
+        services = [service.strip() for service in f.readlines()]
+
+    # 1. Disable services no longer in the service list
+    res = subprocess.run(['systemctl', 'show', 'scion-*@*.service', '--property=Id'], stdout=subprocess.PIPE)
```

Comment:
Can we instead list the dependencies of `scionlab.target`? Something like `systemctl list-dependencies --plain scionlab.target`

Reply:
There is a bug in Ubuntu 16.04 where the dependencies are shown twice. If we care about 16.04 we should remember that bug.

Reply:
Taken care of.
Done